src/utils/searchUtils.ts

```diff
@@ -321,3 +333,7 @@ export function mapQueryToSearchParams({
     },
   }
 }
+
+export function hasOneResult(results: SearchResultsResponse | Error): boolean {
+  return !(results instanceof Error) && results?.results?.totalResults === 1
+}
```

Comment:
I thought writing this function would free me from type checking elsewhere, but sadly no

Reply:
Yea I struggle with this in typescript. You might be able to typecast a la `results as SearchResultsResponse`.

Reply:
Good tip. The typecast worked. But not needed now that the check is happening in a single util function